codeSnippets/snippets/tutorial-full-stack-task-manager/composeApp/src/commonMain/kotlin/network/TaskApi.kt

```diff
@@ -0,0 +1,41 @@
+import io.ktor.client.HttpClient
+import io.ktor.client.call.body
+import io.ktor.client.plugins.contentnegotiation.ContentNegotiation
+import io.ktor.client.plugins.defaultRequest
+import io.ktor.client.request.*
+import io.ktor.http.ContentType
+import io.ktor.http.contentType
+import io.ktor.serialization.kotlinx.json.json
+import kotlinx.serialization.json.Json
+
+class TaskApi {
+    private val httpClient = HttpClient {
+        install(ContentNegotiation) {
+            json(Json {
+                encodeDefaults = true
+                isLenient = true
+                coerceInputValues = true
+                ignoreUnknownKeys = true
+            })
+        }
+        defaultRequest {
+            host = "1.2.3.4"
```

Comment:
Why this host?

Reply:
It's just a placeholder. It should be replaced with the user's IP address (as mentioned in Step 4 in [Create the client](https://ktor.shared.aws.intellij.net/docs/full-stack-development-with-kotlin-multiplatform.html#create-client)).